packages/x-telemetry/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "@mui/x-telemetry",
+  "version": "7.12.0",
+  "description": "MUI X Telemetry",
+  "author": "MUI Team",
+  "main": "src/index.ts",
+  "license": "SEE LICENSE IN LICENSE",
+  "bugs": {
+    "url": "https://github.com/mui/mui-x/issues"
+  },
+  "homepage": "https://mui.com/x/introduction/licensing/",
+  "publishConfig": {
+    "access": "public",
+    "directory": "build"
+  },
+  "scripts": {
+    "typescript": "tsc -p tsconfig.json",
+    "build": "pnpm build:modern && pnpm build:node && pnpm build:stable && pnpm build:types && pnpm build:copy-files && chmod +x build/scripts/cli.js",
+    "build:modern": "node ../../scripts/build.mjs modern",
+    "build:node": "node ../../scripts/build.mjs node",
+    "build:stable": "node ../../scripts/build.mjs stable",
+    "build:copy-files": "node ../../scripts/copyFiles.mjs && node ./scripts/addPackageScripts.js",
+    "build:types": "node ../../scripts/buildTypes.mjs",
+    "prebuild": "rimraf build tsconfig.build.tsbuildinfo"
+  },
+  "bin": {
+    "mui-x-telemetry": "scripts/cli.js"
+  },
+  "packageScripts": {
+    "postinstall": "node ./scripts/postinstall.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/mui/mui-x.git",
+    "directory": "packages/x-telemetry"
+  },
+  "dependencies": {
+    "@babel/runtime": "^7.24.7",
+    "@mui/utils": "^5.15.20",
+    "node-machine-id": "^1.1.12",
+    "is-docker": "^2.2.1",
+    "conf": "^5.0.0",
+    "ci-info": "^4.0.0",
+    "yargs": "^17.7.2"
```

Comment:
https://packagephobia.com/result?p=yargs <1MB

Should it be a dev dependency?

```suggestion
```

Reply:
It's not on the main bundle and we use it only for cli